substra/sdk/client.py

```diff
@@ -617,197 +549,173 @@ def add_compute_plan(
         return self._backend.add(spec, exist_ok=False, spec_options=spec_options)
 
     @logit
-    def get_algo(self, key):
+    def get_algo(self, key: str) -> models.Algo:
         """Get algo by key."""
         return self._backend.get(schemas.Type.Algo, key)
 
     @logit
-    def get_compute_plan(self, key):
+    def get_compute_plan(self, key: str) -> models.Algo:
```

Comment:
This looks like it returns a `models.Algo`, is that intentional? Same for the lines below.

Reply:
too much copy-paste 👀  thanks !